Doc/library/sched.rst

```diff
@@ -66,17 +65,17 @@ Scheduler Objects
 
 .. method:: scheduler.enterabs(time, priority, action, argument=(), kwargs={})
 
-   Schedule a new event. The *time* argument should be a numeric type compatible
-   with the return value of the *timefunc* function passed  to the constructor.
+   Schedule a new event. The *time* argument is a numeric type compatible
+   with the return value of the *timefunc* function passed to the constructor.
    Events scheduled for the same *time* will be executed in the order of their
    *priority*. A lower number represents a higher priority.
 
    Executing the event means executing ``action(*argument, **kwargs)``.
    *argument* is a sequence holding the positional arguments for *action*.
-   *kwargs* is a dictionary holding the keyword arguments for *action*.
+   *kwargs* is a :class:`dict` holding the keyword arguments for *action*.
```

Comment:
I don't think it's necessary to add this markup.  If someone is reading about the `sched` module, then the word dictionary should be enough for them to understand the meaning.